assets/scss/core/_typography.scss

```diff
@@ -107,18 +107,20 @@ small {
   line-height: $dp-font-small-lineheight;
 }
 
-.link-green {
+a {
   color: $dp-color-green;
 
   &:hover {
     color: $dp-color-darkgreen;
   }
 }
 
-.cursive {
-  font-family: 'hand-of-sean';
-}
+.dp-text {
+  &--cursive {
+    font-family: $dp-font-family-handofsean;
+  }
 
-.text-italic {
-  font-style: italic;
+  &--italic {
```

Comment:
Is it a breaking change?

Reply:
Yes, there were two similar classes, what I did was join them and create modifiers

Reply:
Nice! do not forgot to apply all these changes to the clients (at this moment only webapp, right)

Also, if it is a breaking change, you should mark the commit as one:

> Una descripción se DEBE proveer después de `BREAKING CHANGE:`, describiendo qué ha cambiado en la API, e.g., _BREAKING CHANGE: environment variables now take precedence over config files._
> https://www.conventionalcommits.org/es/v1.0.0-beta.3/#especificaci%C3%B3n